docker/nginx/conf.d/default.conf

```diff
@@ -1,12 +1,13 @@
 server {
-    root /srv/api-platform/web;
+    root /srv/app/web;
+    resolver 127.0.0.11;
```

Comment:
`Docker version 17.03.1-ce, build c6d412e`

Removed all images and containers and trying again, will report back with any errors 👍